automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -146,10 +148,21 @@ def satellite6_capsule_upgrade(admin_password=None):
         print('Wrong Capsule Version Provided to upgrade to. '
               'Provide one of 6.1, 6.2')
         sys.exit(1)
+    if not os.environ.get('CAPSULE_SUBSCRIPTION') and not os.environ.get(
+            'CAPSULE_AK'):
+        print('One of Env. Var CAPSULE_SUBSCRIPTION or CAPSULE_AK should be'
```

Comment:
What about making the message a little bit more clear? For example `Neither CAPSULE_SUBSCRIPTION nor CAPSULE_AK environment variables was defined. Define at least one of them in order to run this task.`